services/sources/dspace_service.py

```diff
@@ -64,8 +64,7 @@ def get_records(self, full_import=False, start_timestamp=None, offset: Optional[
 
     def parse_record(self, record):
         try:
-            record = self.source_mapping(record, self.OAI_NAMESPACES, self.constants)
-            record.applyMapping()
+            record = self.source_mapping(record, self.OAI_NAMESPACES)
```

Comment:
May just want to update this lhs variable name to parsed_record or the like to distinguish between record and parsed_record